src/System.CommandLine.Hosting/InvocationLifetime.cs

```diff
@@ -18,8 +23,8 @@ public class InvocationLifetime : IHostLifetime
 
         public InvocationLifetime(
             IOptions<InvocationLifetimeOptions> options,
-            IHostingEnvironment environment,
-            IApplicationLifetime applicationLifetime,
+            IHostEnvironment environment,
+            IHostApplicationLifetime applicationLifetime,
```

Comment:
These arguments are expected to be passed via DI, so the change is not expected to affect consumers of the library.